types/googlemaps/reference/info-window.d.ts

```diff
@@ -1,78 +1,208 @@
 declare namespace google.maps {
+    interface InfoWindowHandlerMap {
+        /**
+         * This event is fired when the close button was clicked.
+         * @see {@link https://developers.google.com/maps/documentation/javascript/reference/info-window#InfoWindow.closeclick Maps JavaScript API}
+         * @see {@link InfoWindow#close}
+         */
+        closeclick: [];
```

Comment:
Why are these arrays?

Reply:
Some events ([example](https://developers.google.com/maps/documentation/javascript/reference/event#MVCArray.remove_at)) have more than one argument. I began to apply a similar approach on the [`Map` events](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4b51763cdda43057b20a89a00b61a1f34d6fe4d5/types/googlemaps/reference/map.d.ts#L2-L133). This was done for consistency.